src/components/message/message.stories.mdx

```diff
@@ -0,0 +1,140 @@
+import { useState } from 'react';
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+
+import Message from '.';
+import Button from '../button';
+
+<Meta
+  title="Message"
+  parameters={{ info: { disable: true }}}
+/>
+
+# Message
+A longer message which stays on screen to read and interpret.
+
+Presents a static message which stays on screen.
+
+Useful for messages which are longer or more important, where the user needs time to interpret them, or might need to refer back to them during an activity.
+
+Various types are available:
+- **Error** - tells the user about a negative outcome that has already happened. Try to focus the message text on the action the user needs to take to be successful, rather than what went wrong.
+- **Info** - gives context or advice to the user where there’s no risk of a negative outcome.
+- **Success** - indicates that an activity was successful. A good example could also present the user with onward options, such as ‘View a list of items’ or ‘Create another’.
+- **Warning** - warns the user about a potential negative outcome that hasn’t happened yet.
+- The Transparent configuration is useful if you’d like the message to be more visually subtle, perhaps in a Dialog.
+
+## Related Components
+
+- Simple positive or negative confirmation? [Try Flash](/?path=/story/flash--default "Flash").
+- Longer, time sensitive message that must be dismissed? [Try Toast](/?path=/docs/design-system-toast--default-story "Toast").
+- Error or warning message that interrupts activity? [Try Alert](/?path=/docs/alert--default-story "Alert").
+
+Import the component and use as shown below:
+
+```jsx
+import Message from "carbon-react/lib/components/message"
```

Comment:
Might as well put a semi-colon on the end of these example imports so the user can just copy and paste and not have to add one in themselves